ode/blueprints/amu/templates/amu/birthdays.html

```diff
@@ -15,13 +15,18 @@ <h2>{{_("Birthdays")}}</h2>
     {% endfor %}
 </ul>
 
-<table class="table table-striped">
+<div class="form-group" style="margin-bottom: 20px;">
```

Comment:
Ich bin gerade in ODE nicht so drin: Eigentlich nutzen wir hier keine inline-Styles, oder? Du könntest da ja eine Klasse `filter-form` o.ä. für nehmen.

Reply:
Ich habe den Style komplett entfernt. Er war nicht wirklich notwendig.